model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -520,18 +504,15 @@ message ProcessBundleSplitResponse {
     // (Required) The grpc read transform reading this channel.
     string ptransform_id = 1;
```

Comment:
It pleases me that the code, the necessary semantics, and the documentation  all line up cleanly. I had a longer comment before that essentially derived the semantics from the Go SDK side changes, partly because I missed this bit of documentation earlier. (The excerpt included here is hidden above the fold)

    // (Required) A definition representing a consumer or producer of this data.
    // If received by a harness, this represents the consumer within that
    // harness that should consume these bytes. If sent by a harness, this
    // represents the producer of these bytes.